packages/toggle-group/src/native/toggle-group-native.tsx

```
@@ -0,0 +1,33 @@
+function Item(props: ItemProps) {
+  if (process.env.NODE_ENV === 'development') {
+    console.log('`Item` from @rn-primitives/toggle-group/native is only supported on native.');
+  }
+  return null;
+}
+
+const useRootContext = () => {
+  throw new Error(
+    'Cannot access the native useRootContext on the web. Please import from `@rn-primitives/toggle-group` or `@rn-primitives/accordion/native`'
+  );
+  return {} as RootContextReturnType;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)